xml/System.Data.Objects.DataClasses/EntityCollection`1.xml

```diff
@@ -463,7 +463,7 @@
           <format type="text/markdown"><![CDATA[  
   
 ## Remarks  
- This method calls the <xref:System.Data.Objects.DataClasses.RelatedEnd.ValidateLoad%2A> method before loading the collection. See this method for a list of possible exceptions.  
+ This method calls the <xref:System.Data.Objects.DataClasses.RelatedEnd.Load*> method before loading the collection. See this method for a list of possible exceptions.  
```

Comment:
... or (if my replacement is accurate) it can all be spelled out like that. It would require being spelled out if the paragraph wants to say something like, "See this method for a list of possible exceptions," since they can't "see" `ValidateLoad` on their own. That's why I went to the trouble to describe the checks in-text.

... or as you say ... kill the paragraph off.